extensions/crossmodel-lang/test/language-server/lexer/cross-model-lexer.test.ts

```diff
@@ -2,14 +2,16 @@
  * Copyright (c) 2023 CrossBreeze.
  ********************************************************************************/
 
-import { describe, expect, test, beforeAll } from '@jest/globals';
+import { describe, expect, test, beforeAll, jest } from '@jest/globals';
 import { EmptyFileSystem } from 'langium';
 import { tokenMatcher } from 'chevrotain';
 
 import { CrossModelLexer } from '../../../src/language-server/lexer/cross-model-lexer';
 import { DEDENT, INDENT } from '../../../src/language-server/lexer/cross-model-indentation-tokens';
 import { createCrossModelServices } from '../../../src/language-server/cross-model-module';
 
+jest.useFakeTimers();
```

Comment:
Why is this needed?

Reply:
I cannot say for sure but it was mentioned in the [Jest documentation for using ESM](https://jestjs.io/docs/ecmascript-modules#differences-between-esm-and-commonjs).

Reply:
I think the code is just an example of using the jest import.

If I read here: https://jestjs.io/docs/timer-mocks#enable-fake-timers I think it's only needed of you need to mock timers.

Reply:
I had some test failures when I removed it in one of the tests so I added it everywhere but I can have a closer look again.

Reply:
Yes, I get the following error in every single test file where I do not have the statement (I left it in one that is why 1 passes):
```
crossmodel-lang: npm ERR! Lifecycle script `test` failed with error: 
crossmodel-lang: npm ERR! Error: command failed 
crossmodel-lang: npm ERR!   in workspace: crossmodel-lang@0.0.1 
crossmodel-lang: npm ERR!   at location: /[...]/crossmodel/extensions/crossmodel-lang 
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm run test
lerna ERR! Test Suites: 9 failed, 1 passed, 10 total
lerna ERR! Tests:       6 passed, 6 total
lerna ERR! Snapshots:   0 total
lerna ERR! Time:        11.764 s
lerna ERR! Ran all test suites.
lerna ERR! npm ERR! Lifecycle script `test` failed with error: 
lerna ERR! npm ERR! Error: command failed 
lerna ERR! npm ERR!   in workspace: crossmodel-lang@0.0.1 
lerna ERR! npm ERR!   at location: /[...]/crossmodel/extensions/crossmodel-lang 
lerna ERR! 
lerna ERR! 
lerna ERR! > crossmodel-lang@0.0.1 test
lerna ERR! > cross-env NODE_OPTIONS=--experimental-vm-modules NODE_NO_WARNINGS=1 jest --passWithNoTests
lerna ERR! 
lerna ERR! 
lerna ERR!     at /[...]/crossmodel/node_modules/lerna/node_modules/execa/index.js:236:11
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^
```

Do you see the same? I can't find any more documentation and the generic one (https://jestjs.io/docs/jest-object#fake-timers) does not give any hints why we need it with ESM.

Reply:
@martin-fleck-at On my machine it complained about the imports, so I added the .js at the end first. Then removed the jest.useFakeTimers(); call and it works on my machine.

With only removing the jest.useFakeTimers(); it indeed gives errors.

Reply:
Interesting but you are right! Thank you!